package.json

```diff
@@ -229,17 +221,16 @@
     "sass-extract": "^2.1.0",
     "sass-lint": "^1.13.1",
     "sass-lint-auto-fix": "^0.21.2",
-    "sass-loader": "^10.0.1",
+    "sass-loader": "^13.2.2",
     "sass-vars-to-js-loader": "^2.1.1",
     "shelljs": "^0.8.4",
     "start-server-and-test": "^1.11.3",
-    "style-loader": "^1.2.1",
-    "terser-webpack-plugin": "^4.1.0",
-    "typescript": "4.0.5",
-    "url-loader": "^4.1.0",
-    "webpack": "^4.44.1",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.0",
+    "style-loader": "^3.3.2",
+    "terser-webpack-plugin": "^5.3.9",
+    "typescript": "4.1.6",
```

Comment:
`Typescript v4.0.5` gives an error below because `@types/express-serve-static-core` uses `TypeScript v4.1` syntax, so I had to update it
> node_modules/@types/express-serve-static-core/index.d.ts:104:68 - error TS1110: Type expected.